nginx/api

```diff
@@ -1,5 +1,8 @@
 server {
+    #listen $PORT;
+    # temporarily hardcode both public and private-serving ports
```

Comment:
If this is temporary can we add a TODO in case we forget? :laughing:

Reply:
Sure - I've actually been told that in docker mode, actually what it does is look for the lowest-number port that's listening and send requests to that.

But it *does* give us `$PORT` anyway, so I feel we may as well use it.